helm/prometheus-rules/templates/platform/honeybadger/alerting-rules/app.rules.yml

```diff
@@ -113,14 +113,14 @@ spec:
       {{- end }}
       for: 40m
       labels:
-        area: managedservices
+        area: platform
         cancel_if_cluster_control_plane_unhealthy: "true"
         cancel_if_cluster_status_creating: "true"
         cancel_if_cluster_status_deleting: "true"
         cancel_if_cluster_status_updating: "true"
         cancel_if_outside_working_hours: "true"
         severity: notify
-        team: atlas
+        team: honeybadger
```

Comment:
This is the appwithoutteamannotation alert because the postmortem https://github.com/giantswarm/giantswarm/issues/30453 has been on your board for a while and you probably care more that apps have a team label than we do

Reply:
I dont think so, I thought it is a monitoring thing that you need that label. If `app-exporter` is the issue of not paing down these labels tho, that is something we need to fix, that is true.

Reply:
From my understandig, it's used to page a team when an app is in an error state (blocked in pending-upgrade for instance) so it's used by app-platform alerts.
It's possible de wanted to use it for monitoring but I think we went in a différent direction. Thanks for the review :)